### PR TITLE
Added example for setting HttpClient.Timeout Property

### LINKED ISSUE #1478: not a single line of example?
if we somehow gather the stats from people coming to this page of what they are *most* likely to be looking for after they have read this documentation page, I am certain we will get this common question: "how do I assign an integer to the Timeout property?", which they are immediately looking to do. My suggestion is that instead of relying on one more google search, add these 2 lines:

```c#
HttpClient c = new HttpClient();
c.Timeout = TimeSpan.FromMinutes(10); // more properties like FromSeconds etc
```

so they 1) google for httpclient timeout property, 2) click the first link in search, 3) glance through the doc for a few seconds, 4) see those 2 lines and Alt+Tab right back to the IDE to continue writing code.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 498a07e4-a734-4344-3e48-e90f80c4e016
* Version Independent ID: 199a48e3-16ed-9fe0-8957-f14f37a17acd
* Content: [HttpClient.Timeout Property (System.Net.Http)](https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpclient.timeout?view=netframework-4.7.2)
* Content Source: [xml/System.Net.Http/HttpClient.xml](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Net.Http/HttpClient.xml)
* Product: **dotnet-api**
* GitHub Login: @karelz
* Microsoft Alias: **ncldev**

### PR DESCRIPTION
**Pull request by HoundThe**
## Summary

Added simple example how to set HttpClient.Timeout Property based on issue 

Fixes #1478

### CODE REVIEW

File: xml/System.Net.Http/HttpClient.xml [hunk 1]
```diff
@@ -2144,6 +2144,13 @@ public class GoodController : ApiController
   
  The same timeout will apply for all requests using this <xref:System.Net.Http.HttpClient> instance.  You may also set different timeouts for individual requests using a <xref:System.Threading.CancellationTokenSource> on a task. Note that only the shorter of the two timeouts will apply.  
   
+ Below is example of setting the <xref:System.Net.Http.HttpClient.Timeout%2A> property.
```
Comment (rpetrusha) [inline xml/System.Net.Http/HttpClient.xml]:
```suggestion
 ## Example
 
 The following example sets the `Timeout` property.
```

File: xml/System.Net.Http/HttpClient.xml [hunk 2]
```diff
@@ -2144,6 +2144,13 @@ public class GoodController : ApiController
...
+  
+```csharp
+HttpClient httpClient = new HttpClient();
+httpClient.Timeout = TimeSpan.FromMinutes(10);
+
```
Comment (rpetrusha) [inline xml/System.Net.Http/HttpClient.xml]:
```suggestion
 ```

File: xml/System.Net.Http/HttpClient.xml [hunk 3]
```diff
@@ -2144,6 +2144,13 @@ public class GoodController : ApiController
   
  The same timeout will apply for all requests using this <xref:System.Net.Http.HttpClient> instance.  You may also set different timeouts for individual requests using a <xref:System.Threading.CancellationTokenSource> on a task. Note that only the shorter of the two timeouts will apply.  
   
+ Below is example of setting the <xref:System.Net.Http.HttpClient.Timeout%2A> property.
+  
+```csharp
+HttpClient httpClient = new HttpClient();
+httpClient.Timeout = TimeSpan.FromMinutes(10);
+ 
+```
```
Comment (rpetrusha) [inline xml/System.Net.Http/HttpClient.xml]: There should be a blank line that follows this end fence.

**Comment by HoundThe** [comment]
Sorry for the high number of commits, I am new to this kind of stuff. I applied your suggestions, I hope I understood them correctly.

**Comment by rpetrusha** [comment]
The extra commits are not a problem, @HoundThe. Your changes look good; thanks for making them. I'll merge your PR now. The example should be live on docs.microsoft.com in the next day or two.

**Comment by HoundThe** [comment]
Thank you

**Pull request merged**